### PR TITLE
User Management API deprecated

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/apidocs-mxsdk/apidocs/user-management-api.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ category: "API Documentation"
 menu_order: 70
 ---
 
+{{% alert type="warning" %}}
+The **User Management** API has been deprecated and will be removed during 2022.
+You are advised not to use this API.
```
Comment (MarkvanMents) [inline content/apidocs-mxsdk/apidocs/user-management-api.md:9]: Is there an alternative API customers could use?

**Comment by MarkvanMents** [comment]
Colleagues on leave until 4 January

File: content/apidocs-mxsdk/apidocs/user-management-api.md (hunk 1 above)
Comment (MarkvanMents) [reply content/apidocs-mxsdk/apidocs/user-management-api.md:9]: @JaapF 
Have you had any response about this?
Let me know if you want to keep this PR open, or whether you want to attack it from a different angle?
THanks
Mark

Comment (JaapF) [reply content/apidocs-mxsdk/apidocs/user-management-api.md:9]: @MarkvanMents 

On slack no response, which could be an indication nobody is using.
I'm now chasing to get usage data from access logs. I guess people are busy.
Let's keep this PR open...thanks for checking

**Comment by MarkvanMents** [comment]
Still no date for deprecation - closing this PR as it will be easy to replicate when necessary.

**Pull request closed without merging**